src/components/markdown-render/h1.js

```diff
@@ -1,8 +1,24 @@
 import React from 'react';
 
 function H1(props) {
+  const { children } = props;
+  const id = children.toString().replace(/\s+/g, '-').toLowerCase();
   return (
-    <h1 className="docs-b-h1 sprk-b-PageTitle sprk-b-TypeDisplayOne sprk-u-mbh sprk-u-Measure" {...props}></h1>
+    <>
+      <span id={id} className="docs-b-Link--anchor" />
```

Comment:
This span thats wrapping the h1 tag needs a closing tag at the end of the heading tag.

Reply:
Resolved this, the intention it to have it NOT wrap the heading.